### ISSUE TITLE
Remove outdated scripts/subpackages

### ISSUE THREAD

**Issue by leewujung** [OSOceanAcoustics/echopop#205]
Many of the scripts and subpackages (folders) are superseded by the methods in the refactored `Survey` class. Let's remove them after #204 is merged so that we keep a clean repo.

**Comment by leewujung** [comment]
Hey @brandynlucca : I think after you finish the majority of #215 it would be a good time to do this. I suggest doing this together with #167, so that all the major disruptions are done at once.

I've created a branch from the current `main` that we can always come back to if we ever need to verify something. But at this point, I suspect the Matlab code is a better reference than the previous Python implementation.

**Comment by brandynlucca** [comment]
This was addressed in PR #227 and can therefore be closed.
